apps/ledger-live-mobile/src/components/DeviceActionModal.tsx

```diff
@@ -40,24 +40,25 @@ export default function DeviceActionModal<Req, Stt, Res>({
   const { t } = useTranslation();
   const showAlert = !device?.wired;
   const [result, setResult] = useState<Res | null>(null);
+  const [forceClose, setForceClose] = useState(false);
 
   const handleModalHide = useCallback(() => {
     if (onModalHide) onModalHide();
     if (onResult && result) {
       onResult(result);
-      setResult(null);
     }
-  }, [onModalHide, onResult, result]);
-
-  const handleClose = useCallback(() => {
     if (onClose && !result) {
```

Comment:
Maybe add a comment to say: "DeviceActionModal onClose prop called in handleModalHide to avoid being it called twice if placed in the QueuedDrawer onClose prop" ?

Reply:
I'll do that

Reply:
Should be good now